### PR TITLE
[POM-292] update priv policies | missing RU

### PR DESCRIPTION
**Pull request by klimtron**

### CODE REVIEW

File: src/app/find-help/reply-offer/reply-offer.component.html [hunk 1]
```diff
@@ -70,14 +70,26 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
...
+          <span *ngIf="currentLang === LanguageCode.en_GB">
+            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci_EN.pdf">
+              <span>{{ "PRIVATE_POLICY" | translate }}</span>
+            </a>
+          </span>
+          <span *ngIf="currentLang === LanguageCode.ru_RU">
+            <--! TODO update private policy with RU ver -->
+            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci_EN.pdf">
+              <span>{{ "PRIVATE_POLICY" | translate }}</span>
+            </a>
```
Comment (mgol) [inline src/app/find-help/reply-offer/reply-offer.component.html:92]: Can we instead set the `href` in the component code and not repeat whole blocks here?

BTW, the key should be named `PRIVACY_POLICY` but that's already there.

Comment (bbartosiak) [reply src/app/find-help/reply-offer/reply-offer.component.html:92]: or
<a target="_blank" [href]=" 'RODO_LINK' | translate ">
  <span>{{ "PRIVATE_POLICY" | translate }}</span>
</a>
but the link will be in the translation key

Comment (mgol) [reply src/app/find-help/reply-offer/reply-offer.component.html:92]: That seems like abusing the translation system, though. :)

Comment (ermenegildo500) [reply src/app/find-help/reply-offer/reply-offer.component.html:92]: How about having a reusable snippet (component?) covering this scenario in reply-offer.component.html and site-footer.component.html (see #197 )

Comment (mgol) [reply src/app/find-help/reply-offer/reply-offer.component.html:92]: @ermenegildo500 that seems like the best option, indeed.

Comment (ermenegildo500) [reply src/app/find-help/reply-offer/reply-offer.component.html:92]: Reusable component under review in #227

**Comment by mgol** [comment]
This PR doesn't seem needed after #227 landed; closing.

**Pull request closed without merging**